src/components/todo.component/todo.component.tsx

```diff
@@ -0,0 +1,171 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable @typescript-eslint/no-unused-expressions */
+/* eslint-disable no-param-reassign */
+/* eslint-disable @typescript-eslint/indent */
+/* eslint-disable jsx-a11y/label-has-associated-control */
```

Comment:
Don't do so

es-lint is here for the reason

Please remove this directives and add them only in the exact lines, when you cannot avoid the problems.

At least these three are the red flags for me

```
/* eslint-disable @typescript-eslint/no-unused-vars */
/* eslint-disable @typescript-eslint/no-unused-expressions */
/* eslint-disable no-param-reassign */
```